src/main/resources/resources/logisim/strings/soc/soc_zh.properties

```diff
@@ -321,40 +321,40 @@ SocPioRisingEdge = 上升沿
 #
 # pio/PioMenu.java
 #
-ErrorCreatingHeaderAndOrCFile = 无法创建头文件和/或c文件！
-ExportC = 导出C模板
-PioMenuBidirFunctionsRemark = /*使用以下功能可以设置或读取双向寄存器。\n*1表示尊重。位是输出，0表示它是输入\n*注意：只使用较低的%s位。\n*/
+ErrorCreatingHeaderAndOrCFile = 无法创建头文件和/或 C 文件！
+ExportC = 导出 C 模板
+PioMenuBidirFunctionsRemark = /*使用以下功能可以设置或读取双向寄存器。\n*1 表示尊重。位是输出，0 表示它是输入\n*注意：只使用较低的 %s 位。\n*/
```

Comment:
Remove spaces before the `\n` occurrences.

Reply:
There's actually no space there, just a characteristic of the Chinese '。' Character